### PR TITLE
Add link to to user post history

### LINKED ISSUE #2059: Forum post count and post history do not exist
On the old userpages there is a post count which also links to a user's post history.

### PR DESCRIPTION
**Pull request by notbakaneko**
Adds a link to the user's post history on their profile page.
fixes #2059 

also pulls in more changes for/from #2590

### CODE REVIEW

File: resources/views/objects/search/_forum_options.blade.php [hunk 1]
```diff
@@ -15,41 +15,62 @@
...
+            <input
+                name="{{ $fields['user'] }}"
+                value="{{ request($fields['user']) }}"
+                class="search-forum-options__input search-forum-options__input--text"
+            >
+        </label>
+    @endif
+
+    {{-- FIXME: remove querystring check? --}}
+    @if ($fields['topicId'] !== false && present(request($fields['topicId'])))
```
Comment (nanaya) [inline resources/views/objects/search/_forum_options.blade.php]: wtf is all this `!== false` check ಠ_ಠ

Comment (nanaya) [reply resources/views/objects/search/_forum_options.blade.php]: I see ಠ_ಠ but why not just null

Comment (notbakaneko) [reply resources/views/objects/search/_forum_options.blade.php]: `null` or `!isset` meant to use the existing value, until I changed it to always force the defaults, so I guess it has no meaning now 💃

File: resources/lang/en/users.php [hunk 2]
```diff
@@ -66,6 +66,12 @@
 
         'here' => 'here', // this is substituted in when generating a link above. change it to suit the language.
     ],
+
+    'posts' => [
+        'title' => ':username\'s posts',
+        'page_description' => 'osu! - Everything you ever wanted to know about :username!',
```
Comment (nanaya) [inline resources/lang/en/users.php]: wat

**Pull request merged**